packages/client/src/Accounts.tsx

```diff
@@ -40,37 +112,84 @@ export default function Accounts(_props: RouteComponentProps) {
 
   return (
     <div>
-      {accounts.map(account => (
-        <section key={account.id}>
-          <header>
-            <h3>{account.email}</h3>
-            {account.loggedIn ? (
-              <Link to={`/accounts/${account.id}/dashboard`}>
-                View Conversations
-              </Link>
-            ) : (
-              <button
-                onClick={() =>
-                  authenticate({ variables: { id: account.id } }).catch(
-                    setError
-                  )
-                }
-              >
-                Log In
-              </button>
-            )}
-          </header>
-        </section>
-      ))}
-      <form onSubmit={onSubmit}>
-        <input
-          type="text"
-          name="email"
-          onChange={e => setEmailValue(e.target.value)}
-          value={emailValue}
-        />
-        <button type="submit">Add Account</button>
-      </form>
+      <div className={classes.appBarSpacer} />
+      <AppBar>
+        <Toolbar>
+          <Typography variant="h6">Poodle</Typography>
+        </Toolbar>
+      </AppBar>
+      <div className={classes.root}>
+        {accounts.map(account => (
+          <section key={account.id}>
+            <header>
+              <Card className={classes.card}>
+                <CardHeader title={account.email} />
+                <CardActions>
+                  {account.loggedIn ? (
+                    <>
+                      <Link
+                        to={`/accounts/${account.id}/dashboard`}
+                        className={classes.link}
+                      >
+                        <Button>View Conversations</Button>
+                      </Link>
+                      <IconButton
+                        aria-label="Delete"
+                        onClick={() =>
+                          deleteAccount({
+                            variables: { id: account.id }
+                          }).catch(setError)
+                        }
+                      >
+                        <DeleteIcon />
+                      </IconButton>
+                    </>
+                  ) : (
+                    <Button
+                      onClick={() =>
+                        authenticate({ variables: { id: account.id } }).catch(
+                          setError
+                        )
+                      }
+                    >
+                      Log In
+                    </Button>
+                  )}
+                </CardActions>
+              </Card>
+            </header>
+          </section>
+        ))}
+      </div>
+      <Fab
+        color="secondary"
+        aria-label="Add Acount"
+        className={classes.fab}
+        onClick={handleClick}
+      >
+        {open ? <CloseIcon /> : <AddIcon />}
+      </Fab>
+      <Modal
+        aria-labelledby="add account modal"
+        aria-describedby="add email account"
```

Comment:
The values for `aria-labelledby` and `aria-describedby` should be DOM ids. `labelledby` should match an id on the modal's title element, and `describedby` should match an id on the descriptive content.